apps/src/code-studio/pd/application_dashboard/cohort_view_table.jsx

```diff
@@ -64,12 +64,6 @@ export class CohortViewTable extends React.Component {
     this.constructColumns();
   }
 
-  showLocked = () => (
```

Comment:
Do we still want to show locked for facilitators? I forget

Reply:
Nm. I suggest keeping this function though and removing the clause about the teacher

Reply:
we do want it for facilitators - I'll put the function back. thanks!

Reply:
ok - but we don't want anyone, including admins, to see locking for teacher applications, so it really is just the clause about whether this is a facilitator cohort table.